config/import-control-test.xml

```diff
@@ -16,4 +16,12 @@
 
   <allow pkg=".*" regex="true" />
 
+  <!-- till https://github.com/checkstyle/checkstyle/issues/6916 -->
+  <subpackage name="checks">
+    <subpackage name="annotation">
+      <allow pkg="org.junit.jupiter.api"/>
+      <disallow pkg="org.junit"/>
```

Comment:
does it make sense to do this on top level, and explicitly allow junit in packages that are not converted yet?

Reply:
No, this is a temporary config. I will add processed packages here so that no one accidentally adds a junit4 test during the migration. Once all tests will use juni5, this section will be removed. Junit4 package annotations will not be available because the junit4 module will be removed from the pom.xml and this section will not be needed any more.

Reply:
I meant something like:
https://github.com/checkstyle/checkstyle/blob/master/config/import-control.xml#L15
https://github.com/checkstyle/checkstyle/blob/master/config/import-control.xml#L70

and it will works same during migration, and in each commit/PR , you will remove `allow` for certain package so so at the end config will self cleanup.

Reply:
done. Now Junit4 globally forbidden and explicitly enabled for every test except tests in `checks/annotation` package. During migration, the allowed package list will shrink until it is empty.